core/common/src/test/java/alluxio/security/authorization/ModeTest.java

```diff
@@ -163,4 +163,46 @@ public void umaskNotIntegerTest() {
         PropertyKey.SECURITY_AUTHORIZATION_PERMISSION_UMASK));
     Mode.getUMask();
   }
+
+  /**
+   * Tests the {@link Mode#setOwnerBits(Mode.Bits)} method.
+   */
+  @Test
+  public void setOwnerBitsTest() {
```

Comment:
We aren't suffixing all of tests with `Test` any more since there is already an `@Test` annotation